apps/omg_performance/coveralls.json

```diff
@@ -1,5 +1,6 @@
 {
   "skip_files": [
-    "test/support"
+    "test/support",
+    "lib/omg_performance"
```

Comment:
hm, why? The coverage drop didn't list `omg_performance` and it should be all quite covered.

Reply:
Isn't the purpose of `omg_performance` to run the tests, not to have test coverage?

Reply:
`omg_performance` is a tool to check aspects of performance, but it as a tool has its own test (such test tells us whether we're able to measure that performance at any commit in `master` - which is very useful). So `omg_performance`'s code is covered by that test and it is coverage we should track.